ui/app/src/components/BrowseFilesDialog/BrowseFilesDialogUI.tsx

```diff
@@ -254,25 +256,34 @@ export function BrowseFilesDialogUI(props: BrowseFilesDialogUIProps) {
             </Paper>
             <Box
               className={classes.cardsContainer}
-              sx={viewMode === 'row' && { display: 'flex !important', flexFlow: 'wrap' }}
+              sx={[viewMode === 'row' && { display: 'flex !important', flexFlow: 'wrap' }]}
             >
               {items
-                ? items.map((item: SearchItem) => (
-                    <MediaCard
-                      viewMode={viewMode}
-                      classes={{
-                        root: clsx(classes.mediaCardRoot, item.path === selectedCard?.path && classes.selectedCard)
-                      }}
-                      key={item.path}
-                      item={item}
-                      selected={multiSelect ? selectedArray : null}
-                      onSelect={multiSelect ? onCheckboxChecked : null}
-                      onPreview={onPreviewImage ? () => onPreviewImage(item) : null}
-                      previewAppBaseUri={guestBase}
-                      onClick={() => onCardSelected(item)}
-                      showPath={true}
-                    />
-                  ))
+                ? items.map((item: SearchItem) => {
+                    const isPreSelected = preselectedPaths.includes(item.path);
```

Comment:
Change `PreSelected` across the board to `Preselected`, single word.
```suggestion
                    const isPreselected = preselectedPaths.includes(item.path);
```